src/crimson/net/SocketMessenger.cc

```diff
@@ -112,6 +112,11 @@ SocketMessenger::connect(const entity_addr_t& peer_addr, const entity_type_t& pe
     });
 }
 
```

Comment:
@cyx1231st I remember you separate messenger shutdown and stop for some reason.  Does this combination stop and shutdown work well for you concern case?  BTW, currently messenger is initialized in main.cc not in osd instance.